orienteer-core/src/main/java/org/orienteer/core/util/LookupResourceHelper.java

```diff
@@ -135,7 +135,9 @@ public DirFileLookuper(File startDir) {
 		}
 		
 		public DirFileLookuper(String path) {
-			this.dir = new File(path);
+			if (!Strings.isEmpty(path)) {
```

Comment:
Not sure that it's good. If path is not setup - this.dir will be still null - and we just pushing NullPointerException one level down.